README.md

```diff
@@ -71,6 +71,10 @@ Run the tests:
 make test
 ```
 
+## Special use cases
+### Fan out Proxies
+Fan out proxies usually rely on response being cacheable to support that you can provide the `Content-Type` header to  `EventSourceResponse`
```

Comment:
I think you meant to say, 

Fan out proxies usually rely on response being cacheable. To support that, you can set the value of `Cache-Control`. For example,

```
[put sample code here that sets cache-control]
```

Reply:
Yep, @paxcodes  I have updated the pull request.